src/repository/user.repository.ts

```diff
@@ -8,6 +9,19 @@ class UserRepository extends BaseRepository {
 		super();
 	}
 
+	async userCheck(email: string) {
```

Comment:
I think better to have return types for these funcs like below but can be done in a separate PR
```suggestion
	async userCheck(email: string): boolean {
```